var/spack/repos/builtin/packages/adlbx/package.py

```diff
@@ -12,12 +12,20 @@ class Adlbx(AutotoolsPackage):
 
     homepage = 'http://swift-lang.org/Swift-T'
     url      = 'http://swift-lang.github.io/swift-t-downloads/spack/adlbx-0.0.0.tar.gz'
+    git      = "https://github.com/swift-lang/swift-t.git"
 
+    version('develop', branch='master')
+    version('0.9.2', 'a7d9e208eb3b49b8bb857562f6bb61bb')
     version('0.9.1', '07151ddef5fb83d8f4b40700013d9daf')
     version('0.8.0', '34ade59ce3be5bc296955231d47a27dd')
 
+    depends_on('exmcutils@develop', when='@develop')
     depends_on('exmcutils@:0.5.3', when='@:0.8.0')
     depends_on('exmcutils', when='@0.9.1:')
+    depends_on('autoconf', type='build')
+    depends_on('automake', type='build')
+    depends_on('libtool', type='build')
+    depends_on('m4', type='build')
```

Comment:
Are these Autotools deps only needed `when='@develop'`?

Reply:
Right- do you have an example of how to express that?

Reply:
Nm, I get it now.

Reply:
Addressed in the latest commit e6f9c76 .